lib/tasks/data.rake

```diff
@@ -12,7 +12,11 @@ namespace :data do
     namespace :audit_data do
       task vacancy_publishing: :environment do
         PublicActivity::Activity.where(key: 'vacancy.publish').map do |audit|
```

Comment:
Just noticed this - is there any reason why we're using `map` instead of `each`?

Reply:
Only that when I don't want to think about the return value I tend to use `map` since I think I read that it's slightly more performant than `each` 🤷‍♂️

Reply:
Ah, interesting. I'm gonna have a dig into that 👍